common/models/user.js

```diff
@@ -80,6 +80,9 @@ module.exports = function(User) {
       cb = options;
       options = undefined;
     }
+
+    cb = cb || app.createPromiseCallback();
```

Comment:
Oh, are you relying on the fact that `test/support.js` is setting up a global `app` variable?